spec/models/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'User' do
+  subject(:user) do
+    User.new(
+      first_name: first_name, last_name: last_name, email: email, password: password,
+      password_confirmation: password_confirmation
+    )
+  end
+
+  let(:first_name)            { Faker::Name.first_name }
+  let(:last_name)             { Faker::Name.last_name }
+  let(:email)                 { Faker::Internet.email }
+  let(:password)              { 'password' }
+  let(:password_confirmation) { 'password' }
+
+  context 'With an empty user' do
+    it '#attr_getters' do
```

Comment:
There is no need to test the existence of attributes in a model! You can remove this example!